src/tempprocessing.f90

```diff
@@ -894,6 +903,32 @@ integer(int32) function GrowingDegreeDays(ValPeriod, FirstDayPeriod, Tbase, &
             DayGDD = DegreesDay(Tbase, Tupper, &
                      TDayMin_local, TDayMax_local, GetSimulParam_GDDMethod())
             GDDays = roundc(ValPeriod * DayGDD, mold=1_int32)
+        else if (GetTemperatureFile() == '(External)') then
+            RemainingDays = ValPeriod
+            i = 1 
+            TDayMin_local = GetTminRun_i(i)
+            TDayMax_local = GetTmaxRun_i(i)
+            DayGDD = DegreesDay(Tbase, Tupper, TDayMin_local, &
+                                        TDayMax_local, &
+                                        GetSimulParam_GDDMethod())
+            GDDays = GDDays + DayGDD
+            RemainingDays = RemainingDays - 1
+
+            do while ((RemainingDays > 0) &
+                        .and. (i<365))
+                        i = i + 1
+                        TDayMin_local = GetTminRun_i(i)
+                        TDayMax_local = GetTmaxRun_i(i)
+                        DayGDD = DegreesDay(Tbase, Tupper, TDayMin_local, &
+                                            TDayMax_local, &
+                                            GetSimulParam_GDDMethod())
+                        GDDays = GDDays + DayGDD
+                        RemainingDays = RemainingDays - 1
+            end do
+
+           if (RemainingDays > 0) then
+                 GDDays = undef_int
```

Comment:
What would that mean? That the cropping period is longer than the simulation period? 

Related to this: when running in GDD, we'll have to be careful in regions where we don't have enough GDDays to complete the crop cycle in one year. I guess we'll just stop the growth and restart a new simulation period, but something to keep in mind when we finally do some science with the code.

Reply:
Yes, we will just stop the growth and restart a new simulation period. 
Should we here include the rewinding and set i back to 1 when GetSimulation_ToDayNr()-GetSimulation_FromDayNr() is reached to catch those cases?